inc/saiswitch.h

```diff
@@ -3000,6 +3000,14 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_AVAILABLE_SYSTEM_VOQS,
 
+    /**
+     * @brief Pre-Ingress ACL stage.
+     *
+     * @type sai_acl_capability_t
+     * @flags READ_ONLY
+     */
+    SAI_SWITCH_ATTR_ACL_STAGE_PRE_INGRESS,
+
```

Comment:
Please add POST_INGRESS stage as well. We are missing it for capability query.

Reply:
IIUC POST_INGRESS attribute is already present?
https://github.com/opencomputeproject/SAI/blob/master/inc/saiswitch.h#L2823
    /**
     * @brief Post Ingress ACL stage.
     *
     * @type sai_acl_capability_t
     * @flags READ_ONLY
     */
    SAI_SWITCH_ATTR_ACL_STAGE_POST_INGRESS,